sql-conditional-logic/sql-conditional-logic_solution.sql

```diff
@@ -0,0 +1,63 @@
+#1
+USE telecom;
+
+#Conditional logic for volume_1
+CREATE TEMPORARY TABLE dsstudent.nicetable
+SELECT id, log_feature, volume,
+	CASE
+		WHEN volume < 100 THEN 'low'
+		WHEN volume > 500 THEN 'large'
+		ELSE 'medium'
+	END volume_1
+FROM log_feature AS lf;
+
+	
+# Selecting columns
+SELECT *
+FROM dsstudent.nicetable;
+
+#Quantity of records for "low, "medium", and "Large
+SELECT volume_1, COUNT(volume_1) AS value_counts
+FROM dsstudent.nicetable
+GROUP BY 1;
+
+#2
+#Conditional logic for HourlyRate_1
+USE hr;
+
+CREATE TEMPORARY TABLE dsstudent.rates
+SELECT EmployeeNumber, HourlyRate,
+	CASE
+		WHEN HourlyRate >= 80 THEN 'high hourly rate'
+		WHEN HourlyRate < 40 THEN 'low hourly rate'
+		ELSE 'medium hourly rate'
+	END HourlyRate_1
+FROM hr.employee;

```

Comment:
You don't need to say hr.employee, you can just say employee since you issued "USE hr"